tests/unittest/_torch/speculative/hw_agnostic/test_dflash_accept_stats.py

```
@@ -0,0 +1,271 @@
+    assert merged["per_request"]["1"]["accepted_draft"] == 2
+    assert merged["per_request"]["2"]["steps"] == 1
+    s = accept_stats.summarize_hist(merged["accepted_draft_hist"])
+    assert s["al"] == pytest.approx(2.0)
+    assert s["ar_per_position"] == pytest.approx([0.5, 0.5])
+
+    with pytest.raises(ValueError):
+        accept_stats.merge_snapshots([])
+
+
+def test_merge_snapshots_shape_mismatch():
+    a = accept_stats.DFlashAcceptStatsRecorder("/tmp", 2, rank=0).snapshot()
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)